library/src/main/res/layout/sd_fill_fab.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2021 Roberto Leinardi.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+<!--    MaxImageSize can't change　programmatically-->
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
```

Comment:
**Reporter**: Android Lint
**Rule**: ContentDescription
**Severity**: WARN
**File**: library/src/main/res/layout/sd_fill_fab.xml L22

Image without &#96;contentDescription&#96;&#10;Missing &#96;contentDescription&#96; attribute on image&#10;Non-textual widgets like ImageViews and ImageButtons should use the &#96;contentDescription&#96; attribute to specify a textual description of the widget such that screen readers and other accessibility tools can adequately describe the user interface.&#10;&#10;Note that elements in application screens that are purely decorative and do not provide any content or enable a user action should not have accessibility content descriptions. In this case, just suppress the lint warning with a tools:ignore&#61;&quot;ContentDescription&quot; attribute.&#10;&#10;Note that for text fields, you should not set both the &#96;hint&#96; and the &#96;contentDescription&#96; attributes since the hint will never be shown. Just set the &#96;hint&#96;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1613406431>*